wasm2c/wasm-rt-impl.c

```diff
@@ -234,6 +234,7 @@ static void os_install_signal_handler(void) {
   }
 
   struct sigaction sa;
+  memset(&sa , '\0', sizeof(sa));
```

Comment:
This isn't strictly related to SIMD but I think it's probably harmless to do here.